src/client_side.cc

```diff
@@ -3304,10 +3307,10 @@ ConnStateData::startPeekAndSplice()
     bio->hold(true);
 
     // Here squid should have all of the client hello message so the
-    // tlsAttemptHandshake() should return false;
+    // tlsAttemptHandshake() should return 0.
```

Comment:
s/0/1/ because tlsAttemptHandshake() is now documented to return `1` on success (not `0`).